scripts/update/update

```diff
@@ -14,6 +14,15 @@ check_dependencies () {
   done
 }
 
+curl_check () {
+  status=$(curl -L --head --silent ${1} | grep HTTP/1.1 | tail -n 1)
```

Comment:
I'm not sure why you're using HTTP/1.1 here, I think just HTTP would also work, in the very unlikely event that GitHub would switch to requiring HTTP2 without announcing it before, this check would always fail, resulting in users being unable to install updates.

Reply:
Good catch. I was thinking that too, but the tricky thing was I wasn't sure if the string `HTTP` could also appear anywhere in the body of the response that would mess up the `200` check in the lines below. Maybe exclusively grep-ing for an expression like `HTTP.*200` as a success indicator could help with this :thinking:

Reply:
```sh
curl_check () {
	curl -LIs -o /dev/null -w "%{http_code}" "$1" | grep -q 200
}
```

Should work too

Reply:
Oh wow this is wizardry, thanks for the suggestion man! :raised_hands: ... just committed here as well